Formula/kontena-cli.rb

```diff
@@ -0,0 +1,68 @@
+class KontenaCli < Formula
+  desc "Command-line client for Kontena container & microservices platform"
+  homepage "https://kontena.io/"
+  url "https://github.com/kontena/kontena.git", :tag => "v1.3.4"
+  head "https://github.com/kontena/kontena.git"
+
+  bottle :unneeded
+
+  depends_on :ruby => "2.1"
+
+  def install
+    # Build the gem from sources and install it
+    (buildpath/"cli").cd do
+      system "gem", "build", "--norc", "kontena-cli.gemspec"
```

Comment:
The gem dependencies need to be vendored. See the `braid` formula.

Reply:
Braid does not build the gem from sources so `--HEAD` does not work.

Would you prefer an alternate `if build.head?` block for HEAD install?

Reply:
i'd just leave out head